cumulusci/core/config/BaseTaskFlowConfig.py

```diff
@@ -12,7 +12,13 @@ def list_infos(infos):
         info = infos[info_name]
         if not info:
             info = {}
-        rv.append({"name": info_name, "description": info.get("description", "")})
+        rv.append(
+            {
+                "name": info_name,
+                "description": info.get("description", ""),
+                "group": info.get("group") or "Other",
```

Comment:
`"group": info.get("group", "")` and let the frontend (cli) decide what the default/unlabelled group name is.